rpm/felix.spec

```diff
@@ -2,7 +2,7 @@
 
 Name:           felix
 Summary:        Project Calico virtual networking for cloud data centers
-Version:        2.2.1
+Version:        2.4.0
```

Comment:
No, I did that yesterday.  This is just because you're looking at a combined change.